stripper/ze_ancient_wrath_v2_test27_m5.cfg

```diff
@@ -0,0 +1,203 @@
+;add timelimit to stage 6 laser boss fight
+add:
+{
+	"classname" "logic_relay"
+	"targetname" "antidelay_lasers"
+	"origin" "100 100 100"
+	"spawnflags" "0"
+	"StartDisabled" "0"
+	"OnTrigger" "commandCommandsay || YOU TOOK TOO LONG ||40-1"
+	"OnTrigger" "playerRunScriptCodeforeach(k,_ in{SetHealth=0}){if(self.GetTeam()==3&&self.GetHealth()>0)EntFireByHandle(self, k,(0).tostring(),0,null,null)}41-1"
```

Comment:
The timing for the laser boss is subject to change. I just set 40 since it makes sense but I might need to make adjustments if so.

Reply:
You probably want to split this into a 2nd logic_relay to avoid possible player confusion. If the boss is killed in the 1 second between 40-41 seconds, then the chat msg would trigger without the slay triggering.